manim/mobject/three_dimensions.py

```diff
@@ -203,16 +207,19 @@ def param_surface(u, v):
                         u_range=[0, 5],
                         )
                     surface_plane.set_style(fill_opacity=1)
-                    surface_plane.set_fill_by_value(axes=axes, colors=[(RED, -0.4), (YELLOW, 0), (GREEN, 0.4)])
+                    surface_plane.set_fill_by_value(axis = "z" ,axes=axes, colors=[(RED, -0.4), (YELLOW, 0), (GREEN, 0.4)])
                     self.add(axes, surface_plane)
         """
+        indices = {"x": 0, "y": 1, "z": 2}
+        ranges = {"x": axes.x_range, "y": axes.y_range, "z": axes.z_range}
```

Comment:
I would suggest the 'axis' parameter to simply take in the numbers 0, 1, 2 rather than a string. This is more inline with other methods such as `.stretch` as well as removes the indices variable while the ranges could just be a list.